src/translator/translator.h

```diff
@@ -225,8 +241,8 @@ class TranslateService : public ModelServiceTask {
 
       auto precison = options_->get<std::vector<std::string>>("precision", {"float32"});
       graph->setDefaultElementType(typeFromString(precison[0])); // only use first type, used for parameter type in graph
-      graph->setDevice(device);
       graph->getBackend()->configureDevice(options_);
+      graph->setDevice(device);
```

Comment:
Is this the right order now? Can you make delta in this file 0 or run regression tests? (Former is easier).

Reply:
My bad. I will fix the order.